src/app/passport/controllers/validate.js

```diff
@@ -14,20 +14,33 @@ class ValidateController extends BaseController {
     };
 
     try {
-      await axios.post(`${API_BASE_URL}${API_VALIDATE_PASSPORT_PATH}`, attributes);
-
-      //TODO: get these values from response when ready
-      req.session.passportParams = {
-        authorizationCode: "THIS-IS-A-FAKE-AUTH-CODE!!!!!!",
-        isValid: true
+      const oauthParams = {
+        ...req.session.authParams,
+        scope: "openid",
       };
 
+      const queryParams = this.getQueryStringParams(oauthParams);
+      const apiResponse = await axios.post(`${API_BASE_URL}${API_VALIDATE_PASSPORT_PATH}${queryParams}`, attributes);
+
+      const code = apiResponse?.data?.code?.value;
+
+      if (!code) {
+        res.status(500);
```

Comment:
All tests we currently have are happy path tests. Did we want to add failure unhappy test cases? for example testing that if there is no code we get status 500

Reply:
We should also add Sonarcloud at some point to keep up to date with the test coverage

Reply:
Yeah good shout, we definitely need some more tests in general for this . I'll try take a look at adding some more today especially around the unhappy path.

And yeah agree we should get Sonarcloud in on this as well